source/articles/2015/bundler-failing-on-el-capitan.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Bundler Failing on El Capitan
+description: How to fix OpenSSL header issues after upgrading to OS X El Capitan
+
+author: Adarsh Pandit
+category: rails
+date: 2015-11-09
+logo: terminal_icon.png
+published: true
+tags: [bundler, ruby on rails, el capitan, openssl, os x, openssl]
+---
+
+After updating to El Capitan,
+installing new gems
+started to fail for what I will call
+"C-like reasons".
```

Comment:
Thanks. I just opted to be less clever here in 6fd8ddd but those are all amazing ideas.